Makefile

```diff
@@ -91,6 +99,14 @@ GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
 golangci-lint:
 	$(call go-get-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/cmd/golangci-lint)
 
+ENVTEST = $(shell pwd)/bin/setup-envtest
+.PHONY: envtest
+envtest: ## Download envtest-setup locally if necessary.
+	$(call go-get-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
```

Comment:
Is it worth pinning `setup-envtest` here to the current `controller-runtime` version we use in `go.mod` (i.e. `v0.11.0`? Maybe we could have it in a variable at the top next to the `ENVTEST_K8S_VERSION` one.

Reply:
Currently kubebuilder generates it with `latest` for all new projects. I don't think we should change this pattern.

Reply:
I see, ok sounds good then 👍